1091. Shortest Path in Binary Matrix/solution.py

```diff
@@ -1,47 +1,79 @@
 class Solution(object):
```

Comment:
Overall comments:
- For coding, you want to indent with 4 spaces rather than 8. 
- Comments should be indented to the same degree as the line of code that follows the comment.

Reply:
By the way, why indent with 4 spaces instead of 8?

Reply:
Ok! I checked and it looks like there's a tab size setting on the GitHub edit UI so I'll set that to 4 from now on. Will change comments as well.